### PR TITLE
Protonvpn cli

### PR DESCRIPTION
**Pull request by ILMostro**

### CODE REVIEW

File: dev-python/pythondialog/metadata.xml [hunk 1]
```diff
@@ -1,7 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!--maintainer-needed-->
+	<maintainer type="person">
+		<email>ilmostro7@gmail.com</email>
```
Comment (epsilon-0) [inline dev-python/pythondialog/metadata.xml:5]: Maybe you want to add a name as well?

File: dev-python/pythondialog/pythondialog-3.5.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+HOMEPAGE="http://pythondialog.sourceforge.net/"
+SRC_URI="mirror://sourceforge/pythondialog//${PV}/python3-${P}.tar.bz2"
+
+LICENSE="LGPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~ia64 ~ppc ~sparc ~x86"
+IUSE="doc examples"
+
+RDEPEND="dev-util/dialog"
+DEPEND="doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )"
```
Comment (epsilon-0) [inline dev-python/pythondialog/pythondialog-3.5.1.ebuild:20]: use `disutils-r1` to use sphinx: https://devmanual.gentoo.org/eclass-reference/distutils-r1.eclass/index.html

File: net-vpn/protonvpn-cli/protonvpn-cli-2.2.2-r1.ebuild [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_6 )
+PYTHON_COMPAT=( python3_{6,7,8} )
```
Comment (epsilon-0) [inline net-vpn/protonvpn-cli/protonvpn-cli-2.2.2-r1.ebuild:6]: this doesn't work with pypy3 ?

File: dev-python/pythondialog/pythondialog-3.5.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} pypy3 )
+
```
Comment (epsilon-0) [inline dev-python/pythondialog/pythondialog-3.5.1.ebuild:7]: `2.2.2` uses `rdepend` for `DISTUTILS_SETUP_TOOLS`, maybe it still is needed?

Did you get a warning when trying to build a package?

File: dev-python/pythondialog/pythondialog-3.5.1.ebuild (hunk 2 above)
Comment (juippis) [reply dev-python/pythondialog/pythondialog-3.5.1.ebuild:20]: Indeed looks like the distutils-r1 function works here.

File: net-vpn/protonvpn-cli/protonvpn-cli-2.2.2-r1.ebuild (hunk 3 above)
Comment (juippis) [inline net-vpn/protonvpn-cli/protonvpn-cli-2.2.2-r1.ebuild:6]: Don't revbump this file. Editing PYTHON_COMPAT in place is fine.

**Pull request closed without merging**